libexec/swiftenv-install

```diff
@@ -193,21 +195,48 @@ build_version() {
 }
 
 find_binary_url_from_api() {
-  vlog "Checking for a URL for the $VERSION on $(get_platform)."
-
-  status_code=$(curl -o /dev/null --silent --head --write-out '%{http_code}' "https://swiftenv-api.fuller.li/versions/$VERSION/binaries/$(get_platform)" || true)
+  vlog "Checking for a URL for Swift $VERSION on $(get_platform)."
+
+  case $(get_platform) in
+    'ubuntu14.04' )
+      SWIFT_BUILD_FOLDER="ubuntu1604"
+      SWIFT_ZIP_FILE="ubuntu14.04"
+      ;;
+    'ubuntu16.04' )
+      SWIFT_BUILD_FOLDER="ubuntu1604"
+      SWIFT_ZIP_FILE="ubuntu16.04"
+      ;;
+    'ubuntu18.04' )
+      SWIFT_BUILD_FOLDER="ubuntu1804"
+      SWIFT_ZIP_FILE="ubuntu18.04"
+      ;;
+    'ubuntu20.04' )
+      SWIFT_BUILD_FOLDER="ubuntu2004"
+      SWIFT_ZIP_FILE="ubuntu20.04"
+      ;;
+    'centoslinux7')
+      SWIFT_BUILD_FOLDER="centos7"
+      SWIFT_ZIP_FILE="centos7"
+      ;;
+    'centoslinux8')
+      SWIFT_BUILD_FOLDER="centos8"
+      SWIFT_ZIP_FILE="centos8"
+      ;;
```

Comment:
Based on previous comment in https://github.com/kylef/swiftenv/pull/183/files#r693112331 the same approach could be used with centos then we can support centos9 etc when it comes out in previous versions without updates to swiftenv.

Reply:
Done